plugin.d.ts

```diff
@@ -0,0 +1,19 @@
+import { FastifyPluginCallback } from 'fastify'
```

Comment:
```suggestion
import { FastifyPlugin } from 'fastify'
```

Reply:
FastifyPlugin is marked as deprecated, and annotated here https://github.com/fastify/fastify/blob/master/types/plugin.d.ts#L29

Reply:
Thanks for the clarification